src/borg/archiver.py

```diff
@@ -2067,14 +2067,21 @@ def do_break_lock(self, args, repository):
             the re module <https://docs.python.org/3/library/re.html>`_.
 
         Path prefix, selector `pp:`
-            This pattern style is useful to match whole sub-directories. The pattern
-            `pp:root/somedir` matches `root/somedir` and everything therein.
+            This pattern style is useful to match whole sub-directories without worrying
+            about escaping special characters. Otherwise, you can just use the default
+            `fm:` or `sh:` styles, which also match whole sub-directories. Unlike those
+            styles, a trailing slash here makes no difference. The pattern `pp:root/somedir`
+            (and `pp:root/somedir/`) matches `root/somedir` and everything therein.
+
+            When used with exclude-norecurse rules, this style is equivalent to `pf:`
+            (except not as efficient).
```

Comment:
For the reasons I already told (scope, not talking about not relevant stuff), I think this just should be:
```
This pattern style is useful to match whole sub-directories
(a trailing slash makes no difference).
```

This is telling all that is in the scope of `pp:`.

I thought about transforming your last sentence to a hint to rather use pf: for exclude-norecurse rules for performance reasons. But then, considering that one usually does not have very many of them, I thought it does not really matter.

The performance point of pf comes when having very many of them (like feeding a full list of all files to borg).

Reply:
A hint about the potential need to escape or quote when using "special characters" (that would otherwise be interpreted/expanded by the shell) should be added to a general section (not to a specific pattern style), if we do not already have that (please check).